pyro/contrib/tracking/extended_kalman_filter.py

```diff
@@ -187,7 +187,7 @@ def update(self, measurement):
         S = H.mm(P).mm(H.transpose(-1, -2)) + R  # innovation cov
 
         K_prefix = self._cov.mm(H.transpose(-1, -2))
-        dx = K_prefix.mm(torch.gesv(dz, S)[0]).squeeze(1)  # K*dz
+        dx = K_prefix.mm(torch.gesv(dz.unsqueeze(1), S)[0]).squeeze(1)  # K*dz
```

Comment:
Just for my understanding - could you comment on why this is failing on the GPU but works fine on CUDA?

Reply:
seems to be a pytorch bug, for some reason on cuda the `gesv` operation will properly handle `(N)` tensors but on cuda they need to be `(N, 1)` tensors.  they both return the same `(N, 1)` output